torch/utils/data/_utils/signal_handling.py

```diff
@@ -32,9 +32,12 @@
 
 import signal
 import threading
-from torch._C import _set_worker_pids, _remove_worker_pids, _error_if_any_worker_fails, _set_worker_signal_handlers  # noqa: F401
 from . import IS_WINDOWS
 
+# Some of the following imported functions are not used in this file, but are to
```

Comment:
Should we import these in those files instead then?

Reply:
no, they are supposed the be exposed under the meaningful namespace `signal_handling.XXX`.